backend/vercel/vercel.go

```diff
@@ -121,6 +122,9 @@ func SetEnvVariable(projectId string, apiKey string, accessToken string, teamId
 	b, err := io.ReadAll(res.Body)
 
 	if res.StatusCode != 200 {
+		if method == "PATCH" {
+			return errors.New(fmt.Sprintf("Could not patch environment variable %s - If this environment variable is already defined in Vercel, please remove it from your Vercel project and retry.", key))
```

Comment:
is there a way our api could remove the env var for them?

Reply:
can't remove it or edit it if it was created manually, we only have permission to update env vars that we create. I think we added some instructions to our Next.js docs to add a `NEXT_PUBLIC_HIGHLIGHT_PROJECT_ID` env var, and if the users do this before installing the integration they'll get this error